themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/DataTable.tsx

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import {ShareResourceModal} from "./ShareResourceModal";
+
+export interface DataTableState {
+    toggleActions: boolean
+}
+
+export interface DataTableProps {
+    data: any[]
+    headers: any[]
+}
```

Comment:
We don't want to use 'any'.  If you run 'npm run lint' the the linter will catch this.